docs/manual/docs/cmd/spfx/project/project-rename.md

```diff
@@ -0,0 +1,39 @@
+# spfx project rename
+
+Renames SharePoint Framework project
+
+## Usage
+
+```sh
+spfx project rename [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-n, --newName <newName>`|New name for the project
+`--generateNewId`|Generate a new solution id for the project
+`--query [query]`|JMESPath query string. See [http://jmespath.org/](http://jmespath.org/) for more information and examples
+`-o, --output [output]`|Output type. `json|text|md`. Default `text`
+`--pretty`|Prettifies `json` output
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
+!!! important
```

Comment:
Important block missing in the command help

Reply:
Added the important block.